testing/trino-testing/src/main/java/io/trino/testing/DistributedQueryRunner.java

```diff
@@ -123,7 +123,7 @@ private DistributedQueryRunner(
                         environment,
                         additionalModule,
                         baseDataDir,
-                        systemAccessControls));
```

Comment:
Admins are not required to configure SAC on workers, but they can. What is the motivation behind this? Just cleanup or there is some issue with this?

Reply:
Cleanup, just noticed this.I don't want to _rely_ on SAC on workers for tests verifying SAC is working properly.

> but they can.

as this is not used for anything, will it actually get loaded?
but that's a different story, right?

Reply:
>  will it actually get loaded?

IIRC it will be somewhere in the guice context.